test/strings/strcat.c

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
No need for this new line. You should simply start with `#include`. Fix this everywhere.